front/lib/resources/app_resource.ts

```diff
@@ -42,47 +38,130 @@ export class AppResource extends ResourceWithVault<App> {
       vaultId: vault.id,
     });
 
-    return new this(AppResource.model, app.get(), vault, []);
+    return new this(App, app.get(), vault);
+  }
+
+  // Fetching.
+
+  private static async baseFetch(
+    auth: Authenticator,
+    options?: ResourceFindOptions<App>
+  ) {
+    const apps = await this.baseFetchWithAuthorization(auth, {
+      ...options,
```

Comment:
We may need to add default includes for vault (and transitively groups) ?

Reply:
It's actually already bundled in `ResourceWithVault`, remember 😉?

https://github.com/dust-tt/dust/blob/82ef5b51894c53f531d1bae88fe57bad14305ad2/front/lib/resources/resource_with_vault.ts#L62

Reply:
I did not see the inheritance, I was on the phone :-)